Assets/Scripts/ECS/Animal/Authoring/AnimalDNAAuthoring.cs

```diff
@@ -40,6 +40,9 @@ private void SetGenes()
             SetComponent((ref BaseSpeed speed) => DNA.NextGene(ref speed.Value));
             SetComponent((ref BaseHearingRange hearingRange) => DNA.NextGene(ref hearingRange.Value));
             SetComponent((ref BaseVisionRange visionRange) => DNA.NextGene(ref visionRange.Value));
+            SetComponent((ref BaseHungerLimit hungerLimit) => DNA.NextGene(ref hungerLimit.Value));
+            SetComponent((ref BaseThirstLimit thirstLimit) => DNA.NextGene(ref thirstLimit.Value));
+            SetComponent((ref BaseMatingLimit matingLimit) => DNA.NextGene(ref matingLimit.Value));
```

Comment:
```suggestion
            SetComponent((ref HungerLimit hungerLimit) => DNA.NextGene(ref hungerLimit.Value));
            SetComponent((ref ThirstLimit thirstLimit) => DNA.NextGene(ref thirstLimit.Value));
            SetComponent((ref MatingLimit matingLimit) => DNA.NextGene(ref matingLimit.Value));
```
Base stats är till för stats som ökar och minskar med en faktor tillfälligt beroende på utomstående faktorer. Exempel på sånt är:
- Sprint => öka speed 100%
- Skadad => sänk speed 50%
- Mörker => sänk vision range 50%

Jag tror inte att det passar till dessa limits. Jag skulle alltså ta bort alla base limits och bara använda de vanliga limit-komponenterna som du har skapat.